multiversx_sdk_cli/tests/test_sign.py

```diff
@@ -0,0 +1,75 @@
+import json
+from pathlib import Path
+from multiversx_sdk_cli.cli import main
+
```

Comment:
The 3 test functions seem very related, and must be run in sequence. Perhaps re-write as a single test function?

Reply:
Actually, I think you can use a `transaction.json` that is crafted in advance, and saved in `testdata` - and it would have all the fields, without `signature`. Then, call `tx sign` on it (that is, no extra `modified` file etc.)